build-java/action.yml

```diff
@@ -82,6 +82,11 @@ runs:
         path: jasper-report
         token: ${{ inputs.token }}
 
+    - uses: actions/checkout@v3
+      if: ${{ inputs.jasperBuild != 'true' }}
+      with:
+        fetch-depth: 0
```

Comment:
Instead of checking out a second time, should this just be added to the above checkout step?